tools/__tasks__/compile/fonts/index.config.js

```diff
@@ -346,18 +346,18 @@ module.exports = [
             {
                 'font-weight': '300',
```

Comment:
Regular weight should always be 400, I think. 300 is for Light. Also weight numbers are missing for Regular weights above for all flavours but ttf hinted.